conans/test/integration/command_v2/list_package_ids_test.py

```diff
@@ -0,0 +1,180 @@
+import re
+
+import pytest
+
+from conans.test.assets.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient, TestServer
+
+
+class TestListPackageIdsBase:
+    @pytest.fixture(autouse=True)
+    def _setup(self):
+        self.client = TestClient()
+
+    def _add_remote(self, remote_name):
+        self.client.servers[remote_name] = TestServer(users={"username": "passwd"}, write_permissions=[("*/*@*/*", "*")])
+        self.client.update_servers()
+        self.client.run("user username -p passwd -r {}".format(remote_name))
+
+    def _create_recipe(self, reference):
+        self.client.save({'conanfile.py': GenConanfile()})
+        self.client.run("create . {}".format(reference))
+
+    def _upload_recipe(self, remote, reference):
+        self.client.save({'conanfile.py': GenConanfile()})
+        self.client.run("export . {}".format(reference))
+        self.client.run("upload --force -r {} {}".format(remote, reference))
+
+
+class TestParams(TestListPackageIdsBase):
+    def test_fail_if_reference_is_not_correct(self):
+        self.client.run("list package-ids whatever", assert_error=True)
+        assert "ERROR: Specify the 'name' and the 'version'" in self.client.out
```

Comment:
We need to improve the UX (not necessarily here). Error message should probably be on the form:

``"whatever" is not a valid reference, provide a reference in the form "name/version"``

Reply:
Agree. This is coming from `ref.py` so it should go in another PR 😉

Reply:
As @franramirez688 says, this is not related to this PR. I just created [an issue](https://github.com/conan-io/conan/issues/9332) to fix this.